packages/theme_tailor/example/lib/theme_extension_in_field.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:example/app_colors.dart';
+import 'package:flutter/material.dart';
+import 'package:theme_tailor_annotation/theme_tailor_annotation.dart';
+
+part 'theme_extension_in_field.tailor.dart';
+
+@tailor
+class $_SomeTheme {
+  static List<Color> appBackground = [AppColors.white, AppColors.black];
+  static List<$_AnotherThemePart> anotherThemePart = [AnotherThemePart.light, AnotherThemePart.dark];
```

Comment:
```suggestion
  @component
  /// OR
  @themeExtension
  static List<$_AnotherThemePart> anotherThemePart = [AnotherThemePart.light, AnotherThemePart.dark];
```
maybe even themeExtension will be better